tests/integration/tables/chrome_extensions.cpp

```diff
@@ -32,17 +32,19 @@ TEST_F(chromeExtensions, test_sanity) {
   // See helper.h for avaialbe flags
   // Or use custom DataCheck object
   // ValidationMap row_map = {
-  //      {"uid", IntType}
-  //      {"name", NormalType}
-  //      {"identifier", NormalType}
-  //      {"version", NormalType}
-  //      {"description", NormalType}
-  //      {"locale", NormalType}
-  //      {"update_url", NormalType}
-  //      {"author", NormalType}
-  //      {"persistent", IntType}
-  //      {"path", NormalType}
-  //}
+  //      {"uid", IntType},
+  //	  {"name", NonEmptyString},
```

Comment:
Nitpick, I know these are only comments but there is inconsistent spacing/indents for `name` and `uid`.